### ISSUE TITLE
Native Mode Not Working

### ISSUE THREAD

**Issue by mar789** [hilts-vaughan/UnlinkMKV-GUI#21]
Upon trying to use the new release and use the Native mode all that happens is the program spams command prompt boxes that stay up for a fraction of a second and then says unlinking has finished.

**Comment by hilts-vaughan** [comment]
Ruh oh. :( You installed Mediainfo into Mkvtoolnix's folder? (The CLI)

**Comment by mar789** [comment]
Yes, tried messing around with all different versions of Mediainfo too. I have since installed everything on my laptop and came out with the same results. Again tried troubleshooting the issue but was unable to find a fix. Still a chance I've got astray in my setup but as far as I'm aware everything is good.

**Comment by mar789** [comment]
Verbose logging also provides me nothing. I tried opening files individually in mediainfo CLI and it works. The CMD boxes that pop up when running native mode seem to have nothing in them.

**Comment by Coveka** [comment]
I'm having the same issue, except when I move the contents of the MediaInfo install into the Mkvtoolnix folder, the application launches but only gives me the screen in the screenshot. Closing the program only gives the same result for all the files in the folder, and when it's gone through all of them, the MKVUnlink gives me the "Unlinking is complete!" message.
![capture73](https://cloud.githubusercontent.com/assets/16465819/20654427/6f2f6bd0-b519-11e6-8958-b49d3ea6ac03.JPG)

**Comment by mar789** [comment]
I had that issue when placing the GUI version of mediainfo in with MKVToolNix. You should be using the CLI version which is the command version.

**Comment by Coveka** [comment]
I just gave what you said a go (CLI) and it just cycles through the files saying "Starting the processing of..." for each of them, while the command windows briefly pop up and close, and then it finishes with no output and the completion message.

**Comment by hilts-vaughan** [comment]
Is your log empty except for "processing..." As well?

**Comment by mar789** [comment]
Yeah, that's all that shows up on mine. I didn't test on my main PC but Perl mode doesn't work on my laptop and I'm certain I have things setup correctly there.

**Comment by hilts-vaughan** [comment]
OK, there's definitely something up. It's working on my Windows machine here -- must be something missing here. 

What version(s) of MKVToolnix for you two? And what system language?

**Comment by mar789** [comment]
Tried both 9.5 and 8.9, currently using 9.5 again.

**Comment by Coveka** [comment]
The latest

**Comment by hilts-vaughan** [comment]
This is embarrassing... The issue is with spaces in the path. I'll deploy a hotfix ASAP, if you want to test in the meanwhile move to a path with no spaces. My windows username is just "Developer", no spaces :(

**Comment by hilts-vaughan** [comment]
The pre-release "UnlinkMKV-GUI 0.2.1" is out which should at least let it begin working now...

**Comment by Coveka** [comment]
It worked, but I still have the olde corrupt video issue. This is for [EG] ZZ Gundam (resub) release

![eg zz_gundam_bd_02_resub 720p 115abf72 mkv_snapshot_01 35_ 2016 11 28_05 45 17](https://cloud.githubusercontent.com/assets/16465819/20656504/2faffa32-b52e-11e6-803f-018de79a8eb3.png)
![eg zz_gundam_bd_02_resub 720p 115abf72 mkv_snapshot_22 35_ 2016 11 28_05 45 47](https://cloud.githubusercontent.com/assets/16465819/20656503/2faf873c-b52e-11e6-82c9-87ec5656ac48.png)
![eg zz_gundam_bd_02_resub 720p 115abf72 mkv_snapshot_24 09_ 2016 11 28_05 46 05](https://cloud.githubusercontent.com/assets/16465819/20656505/2fb1896a-b52e-11e6-8449-7b358c63161e.png)

**Comment by hilts-vaughan** [comment]
Thanks. I'll take a look at that release. I suspect playing back the segmented file will have issues, too if it's like the other release I tried out...

**Comment by hilts-vaughan** [comment]
@Coveka Could you open a seperate for each release you find that does not work? I'll look into them.

**Comment by Coveka** [comment]
Sure, so far I've only tried [EG] ZZ Gundam Resub, so I'll open a thread for that

**Comment by mar789** [comment]
I'm having the same issues on the same episodes as you, if I figure anything out I will let you know.

**Comment by secondsabre** [comment]
Funnily enough, I had this same issue with the same series/encode (ZZ Gundam [EG]) a couple versions back.  I haven't tried the new release, but it's interesting that the same fileset is still having the same problem. Maybe it's just cursed.  =P

https://github.com/hilts-vaughan/UnlinkMKV-GUI/issues/5#issuecomment-146005733
